tests/interop/test_validate_layer_0_keyloak.py

```diff
@@ -0,0 +1,166 @@
+import logging
```

Comment:
@yukinchan Can you rename this file `test_validate_layer_0_keyloak.py` -> `test_validate_layer_0_keycloak.py`

keycloak is misspelled